### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by RHeynsZa** [cypress-io/instrument-cra#228]
Any feeedback from the cypress team? Multiple open issues and PRs, no feedback on any of those, stagnant updates

**Comment by chris-jackson-actionqa** [comment]
@bahmutov ?

**Comment by bahmutov** [comment]
Nope. I use it whenever I need. But since I'm not a cypress team member I cannot help merging and resolving stuff

**Comment by chris-jackson-actionqa** [comment]
> 

Ah, sorry.  I thought you were on their team. :) I guess we need to fork this in order to get the fixes.
